src/features/klant/KlantZoeker.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <form class="search-bar" @submit.prevent="handleSearch">
+    <label
+      ><span>Klanten zoeken</span>
+      <input
+        type="search"
+        placeholder="Zoek op e-mailadres of telefoonnummer"
+        v-model="currentSearch"
+        @search="handleSearch"
+    /></label>
+    <button title="Zoeken">
+      <span>Zoeken</span><utrecht-icon-loupe model-value />
+    </button>
+  </form>
+
+  <section v-if="searchQuery" class="search-section">
+    <simple-spinner v-if="klanten.loading"></simple-spinner>
+    <template v-if="klanten.success">
+      <klanten-overzicht
+        :klanten="klanten.data.page"
+        @klant-selected="emitKlantSelected"
+      />
+      <pagination
+        class="pagination"
+        :pagination="klanten.data"
+        @navigate="navigate"
+      />
+    </template>
+    <application-message
+      v-if="serviceResult?.error"
+      messageType="error"
+      message="Er is een fout opgetreden"
+    ></application-message>
+  </section>
+
+  <h2>test data</h2>
+  <p>0612345789<br />test@conduction.nl</p>
```

Comment:
telnr en email als placeholder opnemen en de todo's weghalen svp

Reply:
Liever geen placeholder. Ik heb het in een title gezet.

Reply:
een placeholder juist wel nuttig in dit veld om duidelijk te maken dat je alleen op email of telnr kan zoeken. anderes even bespreken

Reply:
dit blokkert meerder pr's dus ik resolve de openstaande comments en maak er losse issues van